SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+The versions of the project that are currently supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 4.x | :white_check_mark: |
+| 3.x | :white_check_mark: |
+| < 3.0 | :x: |
+
+## Reporting a Vulnerability
+
+You can report a vulnerability sending us an email at material.ui.org@gmail.com.
```

Comment:
```suggestion
You can report a vulnerability by contacting us by email at material.ui.org@gmail.com.
```

Reply:
Maybe `by contacting us via email` to reduce overuse of `by`